src/core/lib/security/context/security_context.h

```diff
@@ -101,7 +102,8 @@ typedef struct {
   grpc_security_context_extension extension;
 } grpc_server_security_context;
 
-grpc_server_security_context* grpc_server_security_context_create(void);
+grpc_server_security_context* grpc_server_security_context_create(
```

Comment:
what about the client side create, does it have the chance of arena allocation as well?

Reply:
I will do client side in a separate PR (or I can do it in this PR after we decide on the correct way to handle server side).